components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -237,22 +221,135 @@ function OrgSettingsForm(props: { org?: OrganizationInfo }) {
 
-            <Heading2 className="pt-12">Workspace Settings</Heading2>
-            <TextInputField
-                label="Default Image"
-                // TODO: Provide document links
-                hint="Use any official Gitpod Docker image, or Docker image reference"
-                placeholder={globalDefaultImage}
-                value={defaultWorkspaceImage}
-                onChange={setDefaultWorkspaceImage}
-                disabled={isLoading || !org?.isOwner}
+            <Heading2 className="pt-12">Workspace Images</Heading2>
+            <Subheading className="max-w-2xl">
+                Choose a default image for all workspaces in the organization.
+            </Subheading>
+
+            <WorkspaceImageButton
+                disabled={!org?.isOwner}
+                image={settings?.defaultWorkspaceImage ?? globalDefaultImage ?? ""}
+                onClick={() => setShowImageEditModal(true)}
             />
 
-            {org?.isOwner && (
-                <Button htmlType="submit" className="mt-4" disabled={!org.isOwner}>
-                    Update Default Image
-                </Button>
+            {showImageEditModal && (
+                <OrgDefaultWorkspaceImageModal
+                    settings={settings}
+                    globalDefaultImage={globalDefaultImage}
+                    onClose={() => setShowImageEditModal(false)}
+                />
             )}
         </form>
     );
 }
+
+function WorkspaceImageButton(props: { image: string; onClick: () => void; disabled?: boolean }) {
+    function parseDockerImage(image: string) {
+        // https://docs.docker.com/registry/spec/api/
+        let registry, repository, tag;
+        let parts = image.split("/");
+
+        if (parts.length > 1 && parts[0].includes(".")) {
+            registry = parts.shift();
+        } else {
+            registry = "docker.io";
+        }
+
+        const remaining = parts.join("/");
+        [repository, tag] = remaining.split(":");
+        if (!tag) {
+            tag = "latest";
+        }
+        return {
+            registry,
+            repository,
+            tag,
+        };
+    }
+
+    return (
+        <InputField disabled={props.disabled} className="w-full max-w-lg">
+            <div className="flex items-center bg-gray-50 dark:bg-gray-800 p-3 justify-between rounded-lg">
+                <div className="flex items-center overflow-hidden" title={props.image}>
+                    <span className="w-5 h-5 mr-1">
+                        <Stack />
+                    </span>
+                    <span className="truncate font-medium text-gray-700 dark:text-gray-200">
+                        {parseDockerImage(props.image).repository}
+                    </span>
+                    &nbsp;&middot;&nbsp;
+                    <span className="flex-none w-16 truncate text-gray-500 dark:text-gray-400">
+                        {parseDockerImage(props.image).tag}
+                    </span>
```

Comment:
**suggeston**: Leaving one more comment after a [relevant discussion](https://gitpod.slack.com/archives/C05H5UQBW6Q/p1695214535403909?thread_ts=1695206392.710679&cid=C05H5UQBW6Q) (internal) with @filiptronicek in case this is trivial to add.

| Default | Changed state | No write access for members |
|-|-|-|
| <img width="512" alt="Frame 1837" src="https://github.com/gitpod-io/gitpod/assets/120486/eeea592e-ac65-429e-95ba-fb63a6817535"> | <img width="512" alt="Frame 1838" src="https://github.com/gitpod-io/gitpod/assets/120486/e7b3477c-c938-4e7d-8c53-6de616b57cf5"> | <img width="512" alt="Frame 1839" src="https://github.com/gitpod-io/gitpod/assets/120486/49a43409-9e52-47b5-90b3-0a0ef9af0695"> |

Reply:
Thanks @mustard-mh!

💭 **thought**: This will make remove the label if you change back to workspace-full.

❗ **issue**: The default image label is also missing for all other orgs when you change it once.

💡 **suggestion**: What do you think of keeping the label as is for all cases, and simply skip the changed state if we don't have this data yet?

| Default | Changed | No permissions |
|-|-|-|
| <img width="512" alt="Frame 1837" src="https://github.com/gitpod-io/gitpod/assets/120486/3bccf2b3-1413-4ab1-9db6-70551049dd73"> | <img width="512" alt="Frame 1838" src="https://github.com/gitpod-io/gitpod/assets/120486/0ee23483-4fb5-40f3-bae5-09caa1413e84"> | <img width="512" alt="Frame 1839" src="https://github.com/gitpod-io/gitpod/assets/120486/a49217d5-ec42-4c17-bbf9-f188fd66bffb"> |

Reply:
I'm not clear with this comment above @gtsiolis 😂, but to explain the latest changes:

- It will show `Default image` when there is no `defaultWorkspaceImage` in `settings`, you can trigger it by save with an empty string
- Once user change `settings.defaultWorkspaceImage` to `gitpod/workspace-full` it's kind the same with default but will not always *default*, since the real default image can be changed, Once real default changed:
    - Those ORGs with `empty` or `undefined` setting will go with new real default 
    - Those ORGs with `gitpod/workspace-full` specified will go with `gitpod/workspace-full`

Reply:
To have two state display the same time like `default image` and `owner required`, yes we can go with it

Reply:
Sorry for the confusion, @mustard-mh.

The question was about having the Default image prefix at all times, as the one visible there will be the default image, instead of using it to highlight that something has not changed.

Also, shouldn't new orgs come with the default workspace-full?

Reply:
> Also, shouldn't new orgs come with the default workspace-full?

@gtsiolis  On Gitpod Cloud right now, yes. 

it's because `undefined`/ `empty string` on `orgSettings` will fallback to a Gitpod instance config -> default image. Our default is [gitpod/workspace-full](https://github.com/gitpod-io/ops/blob/99af27a28d0f7afa7b420c67d56119857ed5d6bb/deploy/production/meta-us02/app/installer-config.yaml#L168) but it can be changed anytime.

Reply:
This section has description `Choose a default image for all workspaces in the organization.` so user should know that it's the default image for their org workspaces.

I changed code to show a list of descrition, but `Default Image` is still calculated by `!settings.defaultWorkspaceImage`

Reply:
> This section has description Choose a default image for all workspaces in the organization. so user should know that it's the default image for their org workspaces.

I believe this can still be confusing, as people don't read on the internet. Having it as a label could help. But maybe I'm running a bit ahead, thinking this section would grow with more image settings. In any case, let's go with any option that makes sense to you, @mustard-mh. We can always come back to this and improve later. 🏓